### PR TITLE
Adds the `verifier` field to the project response

### PR DESCRIPTION
**Pull request by pcothenet**
### What

- Adds the `verifier` field to the project response

### SDK Release Checklist

- [ ] Have you added an integration test for the changes?
- [ ] Have you built the gem locally and made queries against it successfully?
- [ ] Did you update the changelog?
- [ ] Did you bump the package version [in the code generator](https://github.com/patch-technology/client-code-generation/blob/main/configs/ruby-config.json#L11-L12)?
- [ ] For breaking changes, did you plan for the release of the new SDK versions and deploy the API to production?

### CODE REVIEW

File: .github/workflows/health_check.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Notify slack
         uses: kpritam/slack-job-status-action@v1
         with:
-          if: failure()
+          if: ${{ failure() }}
```
Comment (pcothenet) [inline .github/workflows/health_check.yml:33]: The Slack notifications were not happening for the health check. Testing to see if this works better.

File: spec/integration/projects_spec.rb [hunk 2]
```diff
@@ -49,7 +49,6 @@
       expect(project.average_price_per_tonne_cents_usd)
         .to be_an_instance_of(Integer)
       expect(project.remaining_mass_g).to be_an_instance_of(Integer)
-      expect(project.state).to be_an_instance_of(String)
```
Comment (pcothenet) [inline spec/integration/projects_spec.rb:52]: The underlying data has changed and the project being picked up doesn't have a state (now that it is optional)

**Review by alianam — APPROVED** [review]

File: .github/workflows/health_check.yml (hunk 1 above)
Comment (kleinjm) [reply .github/workflows/health_check.yml:33]: hmm shouldn't need to do it this way but worth a shot

File: spec/integration/projects_spec.rb (hunk 2 above)
Comment (kleinjm) [reply spec/integration/projects_spec.rb:52]: no test added for verifier though?

**Review by kleinjm — APPROVED** [review]

**Pull request merged**